Src/BootCamp.Chapter/Program.cs

```diff
@@ -3,12 +3,65 @@
 using System.IO;
 using Console = System.Console;
 
+
+
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
+           
+            Lesson3.Demo();            
+        }
+        
+        
+        
+        public static string GetStringInput(string prompt)
+        {
+            Console.WriteLine(prompt);
+            return Console.ReadLine();;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845599)